pkg/importer/registry-datasource.go

```diff
@@ -191,10 +191,20 @@ func collectCerts(certDir, targetDir, targetPrefix string) error {
 		return err
 	}
 	for _, obj := range objects {
-		if !strings.HasSuffix(obj.Name(), ".crt") {
+		fileName := obj.Name()
+		hasSuffix := func(suffix string) bool {
+			return strings.HasSuffix(fileName, suffix)
+		}
+		if !hasSuffix(".crt") && !hasSuffix(".pem") {
 			continue
 		}
-		if err := util.LinkFile(filepath.Join(certDir, obj.Name()), filepath.Join(targetDir, targetPrefix+obj.Name())); err != nil {
+		targetName := ""
+		if hasSuffix(".pem") {
+			targetName = targetPrefix + strings.TrimSuffix(fileName, ".pem") + ".crt"
```

Comment:
@awels This has to be done because of containers/image library only allowing .crt's, but is pretty hacky
I'd say if we are documenting this limitation, we could just change https://github.com/kubevirt/containerized-data-importer/blob/97fb3ca9222027032f284aeaa98cedc745921242/pkg/common/common.go#L80
And later decide if we want to be the compatibility layer here